go.mod

```diff
@@ -44,14 +44,14 @@ require (
 	golang.org/x/crypto v0.14.0
 	golang.org/x/mod v0.13.0
 	golang.org/x/text v0.13.0
-	k8s.io/api v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/client-go v0.27.2
-	k8s.io/component-base v0.27.2
-	k8s.io/klog/v2 v2.90.1
+	k8s.io/api v0.28.2
```

Comment:
CAPI `v1.5.2` is still using `k8s.io/api v0.27.2`. 
This is probably not a welcome upgrade at the moment. 
/hold